bitnami/mlflow/templates/run/dep-job.yaml

```diff
@@ -181,9 +181,8 @@ spec:
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.run.livenessProbe "enabled") "context" $) | nindent 12 }}
             exec:
               command:
-                - python
-                - -c
-                - import mlflow; mlflow.__version__
+                - mlflow
+                - --version
```

Comment:
How does executing `--version` command ensure the mlflow process is alive?

Reply:
We don't have a process to check in this kind of container. I am checking the same differently. The same happens with the `readinessProbe`.